### PR TITLE
Adds xref to understanding compute resources section to configuring cluster memory assembly

### LINKED ISSUE #32986: [enterprise-4.7] Issue in file nodes/clusters/nodes-cluster-resource-configure.adoc
### Which section(s) is the issue in?
Understanding managing application memory

### What needs fixing?
`It is recommended to fully read the overview of how OpenShift Container Platform manages Compute Resources before proceeding.`
Suggest to provide a link  to that overview!

### PR DESCRIPTION
**Pull request by stevsmit**
Small GH issue I thought I could tackle. 

GH: https://github.com/openshift/openshift-docs/issues/32986

Preview: https://deploy-preview-36149--osdocs.netlify.app/openshift-enterprise/latest/nodes/clusters/nodes-cluster-resource-configure?utm_source=github&utm_campaign=bot_dp#nodes-cluster-resource-configure-about_nodes-cluster-resource-configure

Will need cherry-picked to 4.6 and 4.7.

### CODE REVIEW

**Comment by maxwelldb** [comment]
@stevsmit Is this valid for 4.9? If so, should it be retargeted to `main`?

File: nodes/clusters/nodes-cluster-resource-configure.adoc
Comment (maxwelldb) [inline nodes/clusters/nodes-cluster-resource-configure.adoc:31]: Is this supposed to exist within "Understanding managing application memory"? If so, it needs to be an H3.

**Review by maxwelldb — DISMISSED** [review]
Link seems to be fine. I have a question about the heading order, though.

File: nodes/clusters/nodes-cluster-resource-configure.adoc
Comment (bergerhoffer) [inline nodes/clusters/nodes-cluster-resource-configure.adoc:31]: Actually, if you're trying to make it look like an additional resources section of the module, then we use the following syntax:

```
.Additional resources
```

File: nodes/clusters/nodes-cluster-resource-configure.adoc
Comment (bergerhoffer) [inline nodes/clusters/nodes-cluster-resource-configure.adoc:32]: .html -> .adoc

**Review by bergerhoffer — COMMENTED** [review]
@stevsmit Adding a few comments I noticed (since I had this still open from your question earlier).

Also, is there a reason that this wouldn't apply to 4.9? I still see that text in 4.9, so I assume that this PR should be based against `main`  (and CP'd to 4.6, 4.7, 4.8, and 4.9) instead of based directly off of the 4.8 branch.

File: nodes/clusters/nodes-cluster-resource-configure.adoc
Comment (stevsmit) [reply nodes/clusters/nodes-cluster-resource-configure.adoc:32]: Ahh crap. I had .adoc before, but then I re-copied the link because it wasn't working correct. I knew that...

File: nodes/clusters/nodes-cluster-resource-configure.adoc
Comment (maxwelldb) [reply nodes/clusters/nodes-cluster-resource-configure.adoc]: Oh, whoops. Andrea's right. Our convention is to make it a  label. 

Not an appropriate a11y practice, but that's beyond the scope of your PR. :)

**Comment by bergerhoffer** [comment]
@maxwelldb Just FYI, I think we decided against using the "changes requested" option when submitting reviews. I think some felt it was too forceful, and it also seems to not clear itself and has to be manually dismissed a lot.

**Comment by maxwelldb** [comment]
> @maxwelldb Just FYI, I think we decided against using the "changes requested" option when submitting reviews. I think some felt it was too forceful, and it also seems to not clear itself and has to be manually dismissed a lot.

@bergerhoffer Is there a paper trail for that? Seems a little restrictive not to use a prominent part of GH--particularly for issues that should hold merging a PR until they're resolved. "I'm not sure if this would be going to the right branches," for example.

**Comment by bergerhoffer** [comment]
> > @maxwelldb Just FYI, I think we decided against using the "changes requested" option when submitting reviews. I think some felt it was too forceful, and it also seems to not clear itself and has to be manually dismissed a lot.
> 
> @bergerhoffer Is there a paper trail for that? Seems a little restrictive not to use a prominent part of GH--particularly for issues that should hold merging a PR until they're resolved. "I'm not sure if this would be going to the right branches," for example.

@maxwelldb there is. From the "Guidelines for OpenShift peer reviewers" doc (I can shoot you a link), it says "When submitting your review, avoid changing the radio button from “Comment” to “Request changes”."

**Comment by maxwelldb** [comment]
@bergerhoffer Commented there just now, but "avoid" doesn't equate to "do not use," IMO. Saving it for issues that absolutely must be addressed prior to merging doesn't seem to run afoul of that.

**Comment by bergerhoffer** [comment]
> @bergerhoffer Commented there just now, but "avoid" doesn't equate to "do not use," IMO. Saving it for issues that absolutely must be addressed prior to merging doesn't seem to run afoul of that.

Sure, but a lot of people are going to have differing opinions about what is "required". Honestly that changes requested thing is more of an annoyance in my opinion than helpful, because it never seems to get cleared even after the author addresses the feedback. Plus those with merge rights can still merge anyways. In general, I think it's viewed as too harsh and unnecessary.

**Comment by maxwelldb** [comment]
@bergerhoffer I find it helpful. ¯\\_(ツ)\_/¯ I would've just toggled it back after clarifying the intended merge targets. 

Nailing down the branches you want a change in is a hard requirement, as far as I can tell.

**Comment by bergerhoffer** [comment]
@maxwelldb I agree that the branch issue is something that needs to be resolved before the PR can be merged, but it's just not part of our workflow to mark the PR as unmergable until certain things have been fixed. So it doesn't make sense for one or two people to use that, but not everyone. That's why it was added to that guidelines doc.

**Comment by maxwelldb** [comment]
@bergerhoffer I'm using a stock GitHub feature, and nothing in our contribution or peer review guidelines say that it's forbidden. 

Gonna make a poll to see what the team thinks.

**Comment by bergerhoffer** [comment]
> @bergerhoffer I'm using a stock GitHub feature, and nothing in our contribution or peer review guidelines says that it's forbidden.
> 
> Gonna make a poll to see what the team thinks.

That's fine to make a poll. But I did just point you to a section in our peer review guidelines that says about not using it, so it's not like it hasn't been discussed and documented.

**Comment by maxwelldb** [comment]
@bergerhoffer If the idea is to communicate "do not," I think that wording should be updated, which should trigger team discussion, anyway.

**Pull request merged**
